option.go

```diff
@@ -243,3 +244,16 @@ func (w withDBClient) config(builder *serverBuilder) {
 func WithDBClient(ci db.ClientInterface) Option {
 	return &withDBClient{ci: ci}
 }
+
+type withObservabilitySender struct {
+	sender observability.Sender
+}
+
+func (w withObservabilitySender) config(builder *serverBuilder) {
+	builder.observabilitySender = w.sender
+}
+
+func WithObsSender(sender observability.Sender, imapErrorType, messageErrorType, otherErrorType int) Option {
```

Comment:
1. Please rename to `WithObservabilitySender()`
2. You're not calling this yet. Waiting for some backend changes?

Reply:
1. ✅ 
2. We wont be calling this in Gluon. It will be used by Bridge so it can pass the observability service down to Gluon.